docs/pages/guide/01-start.md

```diff
@@ -1,6 +1,122 @@
 ---
 layout: guide
 group: guide
-title: Getting Started
+title: Product Overview
 ---
-## Welcome to Navi
+
+## <span class="c5">Yavin overview</span>
+
+<img src="assets/images/Yavin_logo.png" width="200"/>
+
+Yavin is a framework for rapidly building custom data applications that
+offers both a UI and an API. In addition, Yavin can be deployed as a
+standalone Business Intelligence tool. It is <span class="c1">scalable
+and secure</span> and it’s <span
+class="c1">intuitive</span> visualization interface allows for <span
+class="c1">fast drill-downs</span> of very large data sets through
+simple interface and visualization blocks. With Yavin, you can easily
```

Comment:
You've already used intuitive in the sentence above (and it is a synonym for easily), I would remove it here.